docs/document/content/user-manual/shardingsphere-proxy/observability/_index.cn.md

```diff
@@ -112,6 +114,28 @@ bin/start.sh
 ```
 正常启动后，可以在 ShardingSphere-Proxy 日志中找到 plugin 的加载信息，访问 Proxy 后，可以通过配置的监控地址查看到 `Metric` 和 `Tracing` 的数据。
 
+### 通过容器环境使用
+
+- 假设本地已存在包含 shardingsphere-agent.jar 和对应 Agent 配置文件的路径为 `./custom/agent`，
+  可通过环境变量 `JVM_OPT` 来配置 ShardingSphere Agent 的使用。
+  以 Docker Compose 的启动环境为例，假设已存在包含 ShardingSphere Proxy 配置文件的文件夹为 `./custom/conf`，
```

Comment:
Shoule `ShardingSphere Proxy` be `ShardingSphere-Proxy`?

Reply:
> Agent 配置文件的路径为 `./custom/agent`

I think the it's not a path of configuration file, it is a path of agent folder?

Reply:
- This is a folder, and the structure of the agent folder is above the document.  Do I need to mark something?

Reply:
Because I noticed that you used the word "configuration file", which may be misleading, because the real "configuration file" is in the `./custom/agent/conf` directory

My personal suggestion here is to describe it as agent directory or agent folder.

Reply:
I think it may be relatively easy to understand the form of a child list. 🧐